Bottom: bf00e8807b5e36adb4cfcd81ce9fe9653047a674
Top:    ba197369d715e175f75a639718362143f68e3d3c
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2012-10-21 11:34:25 +0100

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index 6ceb7ee..a6e003d 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2610,7 +2610,12 @@ IsPolygonInPolygon (PolygonType *P1, PolygonType *P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
